IFC 4.3/Alignment_3/readme.md

```diff
@@ -0,0 +1,58 @@
+
+## Alignment_3
+
+### Intent
+
+Automatically generated IFC file from a corresponding test file in Civil 3D. 
+
+Converting the data given by the API of Civil into the IFC schema. Exports an 
+alignment containing a horizontal and a vertical alignment segments.
+
+Information for the alignment:
+
+Horizontal:
+	- Length Horizontal: 72.355 [m]
+	- Angle from X axis: 0.462 [rad]
+
+Vertical:
+	- Length along Horizontal: 72.349 [m]
+	- Gradient: -1.22 [%]
+
+
+### Prerequisites
+
+This scenario builds upon following other scenarios:
+- ProjectSetup_1
+
+
+
+### Content
+
+This scenario covers the following concepts and/or IFC entities:
+- Semantic alignment concepts
+	- `IfcAlignment`
+	- `IfcAlignmentHorizontal`
+	- `IfcAlignmentVertical`
+	- `IfcAlignmentHorizontalSegment`
+	- `IfcAlignmentVerticalSegment`
+- Geometric alignment concepts
+	- `IfcGradientCurve`
+	- `IfcCompositeCurve`
+	- `IfcCurveSegment`
+	- `IfcLine`
+- Placement concepts
+	- `IfcAxis2PlacementLinear`
+	- `IfcPointByDistanceExpression`
```

Comment:
@pjanck not sure if this is still up-to-IFC4x3RC2-date but might be important for your final report. please close and resole this comment if everything is stated what you wish to have in this UT description